### ISSUE TITLE
Show partitions tables in the schema which is setted a different name from parent 

### ISSUE THREAD

**Issue by djr08** [dbeaver/dbeaver#18819]
### Description

Step1: create table in schema1
CREATE TABLE  _schema1_.test(col1 varchar(2) Not NULL,Tdate date NOT NULL) PARTITION BY RANGE (Tdate);

Setp2: create partition in schema2

create table _schema2_.test_202201 partition of _schema1_.test for values from ('2022-01-01') to ('2022-02-01');
create table _schema2_.test_202202 partition of _schema1_.test for values from ('2022-02-01') to ('2022-03-01');

Setp3:check the result NG
in schema1, I can see table test,and under the test I can see test_202201 and test_202202 under the folder partition.
but in schema2, I can not see test_202201 test_202202 under the table folder.

![image](https://user-images.githubusercontent.com/122240611/213603055-96a599c4-dd9e-43cd-af6c-837e680cd0ca.png)

### DBeaver Version

verson21.2.1.202109181446

### Operating System

windows

**Comment by E1izabeth** [comment]
What database do you use?

**Comment by LonwoLonwo** [comment]
Hello @djr08 

DBeaver shows partitions as child objects of the table. Your parent table is in schema1, and DBeaver shows partitions to this table.

What is the reason to keep partitions from schema1 in another schema?

**Comment by djr08** [comment]
> What database do you use?

PostgreSQL

**Comment by djr08** [comment]
> Hello @djr08
> 
> DBeaver shows partitions as child objects of the table. Your parent table is in schema1, and DBeaver shows partitions to this table.
> 
> What is the reason to keep partitions from schema1 in another schema?
----------------------------------------------------------------
Thank you for your answer.
I just want to campare these two ways use the same table name, and want to make sure they are the same useful.

In fact, **_before Postgre(9.3)_** it support only one way to create partition:
step1 create table test in  schema1 
step2 create table test_202201 in schema2 in INHERITS (schema1.test).
And at that time , I can see both test and test_202201 as table object.

But **_after Postgre(10)_** it supprot another way to create partition:
Step1, create table test in  schema1 PARTITION BY RANGE (date);
Step2, create table test_202201 in schema2 partition of schema1 for values from (20220101) to (20220201);
then, I can just see the test as a table object in schema1, but test_202201 as partition,maybe it just as a part of table in logic，
but from DDL "create table " , I think it should also be treated as a separate table object in schema2

**Comment by LonwoLonwo** [comment]
It is still not clear - what is the reason for keeping partitions in another schema? And how can we show them in schema2, if, for now, we are showing partitions only as child objects?

But ok.

**Comment by djr08** [comment]
> It is still not clear - what is the reason for keeping partitions in another schema? And how can we show them in schema2, if, for now, we are showing partitions only as child objects?
> 
> But ok.

------------------------------------------------------------------------------------------------------------------
The reason is that i use a5m2（DB tools client ) to see the table object,there is a long list(10years tables name) in schema1 when parent table and subtable are both in it. So i use another schema like schema2 just reference subtables(monthly).

Then I change to use dbeaver to see them,and I found it is kindly to see table and partition because it was sorted by folder。
But from dbeaver I found that in schema2 I can not see the table object(monthly).
